library/std/src/process.rs

```diff
@@ -1029,6 +1029,84 @@ impl Command {
     pub fn get_current_dir(&self) -> Option<&Path> {
         self.inner.get_current_dir()
     }
+
+    /// Convenience wrapper around [`Command::status`].
+    ///
+    /// Returns an error if the command exited with non-zero status.
+    ///
+    /// # Examples
+    ///
+    /// ```no_run
+    /// # #![feature(command_easy_api)]
+    /// use std::process::Command;
+    ///
+    /// let res = Command::new("cat")
+    ///                   .arg("no-such-file.txt")
+    ///                   .run();
+    /// assert!(res.is_err());
+    /// ```
+    #[unstable(feature = "command_easy_api", issue = "none")]
+    pub fn run(&mut self) -> io::Result<()> {
+        let status = self.status()?;
+        self.check_status(status)
+    }
+
+    /// Convenience wrapper around [`Command::output`] to get the contents of
+    /// standard output as [`String`].
+    ///
+    /// The final newline (`\n`) is stripped from the output. Unlike
+    /// [`Command::output`], `stderr` is inherited by default.
+    ///
+    /// Returns an error if the command exited with non-zero status or if the
+    /// output was not valid UTF-8.
+    ///
+    /// # Examples
+    ///
+    /// ```no_run
+    /// # #![feature(command_easy_api)]
+    /// use std::process::Command;
+    /// let output = Command::new("git")
+    ///                      .args(["rev-parse", "--short", "1.0.0"])
+    ///                      .read_stdout()?;
+    ///
+    /// assert_eq!(output, "55bd4f8ff2b");
+    /// # Ok::<(), std::io::Error>(())
+    /// ```
+    #[unstable(feature = "command_easy_api", issue = "none")]
+    pub fn read_stdout(&mut self) -> io::Result<String> {
```

Comment:
Bikeshed: I suggest `get_stdout` for the name?  To me, `read_stdout` fails to suggest that it also waits for termination.